sdk/azidentity/chained_token_credential.go

```diff
@@ -17,16 +17,14 @@ import (
 
 // ChainedTokenCredentialOptions contains optional parameters for ChainedTokenCredential.
 type ChainedTokenCredentialOptions struct {
-	// RetrySources configures how the credential uses its sources.
-	// When true, the credential will always request a token from each source in turn,
-	// stopping when one provides a token. When false, the credential requests a token
-	// only from the source that previously retrieved a token--it never again tries the sources which failed.
+	// RetrySources configures how the credential uses its sources. When true, the credential always attempts to
+	// authenticate through each source in turn, stopping when one succeeds. When false, the credential authenticates
+	// only through this first successful source--it never again tries the sources which failed.
 	RetrySources bool
 }
 
-// ChainedTokenCredential is a chain of credentials that enables fallback behavior when a credential can't authenticate.
-// By default, this credential will assume that the first successful credential should be the only credential used on future requests.
-// If the `RetrySources` option is set to true, it will always try to get a token using all of the originally provided credentials.
+// ChainedTokenCredential links together multiple credentials and tries them sequentially when authenticating. By default,
+// it tries all the credentials until one authenticates, after which it always uses that credential.
```

Comment:
Should the second sentence of this be removed since it contradicts what the ChainedTokenCredentialOption is saying for 'RetrySources'? Or am I misreading it?